attentive-android-sdk/src/main/java/com/attentive/androidsdk/events/Purchase.java

```diff
@@ -0,0 +1,22 @@
+package com.attentive.androidsdk.events;
+
+public class Purchase extends Event {
+    private double amount;
```

Comment:
I think we send should the amount as a string & convert to a number value later on the backend